TCAT/Controllers/AllStopsTableViewController.swift

```diff
@@ -19,7 +19,6 @@ class AllStopsTableViewController: UITableViewController {
     var sectionIndexes: [String: [BusStop]]!
     var sortedKeys: [String]!
     var unwindAllStopsTVCDelegate: UnwindAllStopsTVCDelegate?
-    var navController: UINavigationController!
```

Comment:
Unused variable found while combing through for instances of UINavigationController